### PR TITLE
pin vtk and occt

### PR DESCRIPTION
**Pull request by looooo**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [ ] Ensured the license file is being packaged.

Closes https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/69

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set version = "2018.05.22" %}
+{% set version = "2018.05.24" %}
```
Comment (jakirkham) [inline recipe/meta.yaml]: Do we want to do the version bump here or were there other things we need to get in first?

cc @conda-forge/core

Comment (scopatz) [reply recipe/meta.yaml]: We can always do another release.  I don't think it is a big issue

**Comment by scopatz** [comment]
Thanks for putting this in @looooo. Was occt supposed to be here?

**Comment by looooo** [comment]
yes I thought it will help for packages depending on occt using the same version. Doesn't it make sense to have it pinned?

**Comment by scopatz** [comment]
Ok, I just didn't see occt in the changeset

**Comment by looooo** [comment]
ah you are right, something went wrong

**Comment by looooo** [comment]
Is there a way to use a newer occt if this PR is accepted and a newer occt is uploaded?
I think the `max_pin: x.x` is absolutely needed for occt to not include some incompatible versions.

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Comment by jakirkham** [comment]
Thanks @looooo :)

**Pull request merged**